packages/core/src/defaultCharacter.ts

```diff
@@ -5,7 +5,7 @@ export const defaultCharacter: Character = {
     username: "eliza",
     plugins: [],
     clients: [],
-    modelProvider: ModelProviderName.LLAMALOCAL,
+    modelProvider: ModelProviderName.OLLAMA,
```

Comment:
@lalalune test `packages/core/src/defaultCharacter.ts`  was failing because of that change to `LLAMALOCAL`. Was that change intentional? I switched back to `OLLAMA`